bot/exts/events/advent_of_code/_cog.py

```diff
@@ -52,13 +60,80 @@ def __init__(self, bot: Bot):
         self.status_task.set_name("AoC Status Countdown")
         self.status_task.add_done_callback(_helpers.background_task_callback)
 
+        self.completer_task.start()
+
+    @tasks.loop(hours=1)
+    async def completer_task(self) -> None:
+        """
+        Every hour check who has completed all 50 stars and give them the completionist role.
```

Comment:
One small note - for consistency with the other docstrings, how about something like
```suggestion
        Checks who has completed all 50 stars and give them the completionist role hourly.
```

Reply:
Updated the docstring 👍